runtime/stdlib/test.go

```diff
@@ -153,6 +151,34 @@ var blockchainBackendInterfaceType = func() *sema.InterfaceType {
 	return interfaceType
 }()
 
+var matcherType = func() *sema.CompositeType {
+	typ, ok := testContractType.NestedTypes.Get(matcherTypeName)
+	if !ok {
+		panic(errors.NewUnexpectedError("cannot find type %s.%s", testContractTypeName, matcherTypeName))
+	}
+
+	interfaceType, ok := typ.(*sema.CompositeType)
+	if !ok {
+		panic(errors.NewUnexpectedError("invalid type for %s. expected interface", matcherTypeName))
+	}
+
+	return interfaceType
+}()
+
+var matcherTestFunctionType = func() *sema.FunctionType {
+	testFunc, ok := matcherType.Members.Get(matcherTestFunctionName)
+	if !ok {
+		panic(errors.NewUnexpectedError("cannot find type %s.%s", matcherTypeName, matcherTestFunctionName))
```

Comment:
```suggestion
		panic(errors.NewUnexpectedError("cannot find member %s.%s", matcherTypeName, matcherTestFunctionName))
```

Reply:
ah the ripple effects 😬 
This was refactored in the follow-up PR: https://github.com/onflow/cadence/pull/1897. I'll fix it there; that should fix it for all places at once.

Reply:
fixed in fbd4f3f490a2e254fbc04b985314b1383abdfccd